_data/versions.yml

```diff
@@ -1350,30 +1350,44 @@ v3.0:
+    
+    - `calicoctl` no longer includes a `config` command. To achieve the equivalent functionality: refer to [Modifying low-level component configurations](https://docs.projectcalico.org/v3.0/reference/calicoctl/commands/#modifying-low-level-component-configurations).
 
     - You can now name [host](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/hostendpoint#endpointport) and [workload](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/workloadendpoint#endpointport) endpoint ports and reference them by name in your [policy rules](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy#ports).
 
     - The new `ApplyOnForward` flag allows you to specify if a host endpoint policy should apply to forwarded traffic or not. Forwarded traffic includes traffic forwarded between host endpoints and traffic forwarded between a host endpoint and a workload endpoint on the same host. Refer to [Using Calico to secure host interfaces](https://docs.projectcalico.org/v3.0/getting-started/bare-metal/bare-metal) for more details.
 
     - Calico now works with Kubernetes network services proxy with IPVS/LVS. Calico enforces network policies with kube-proxy running in ipvs mode for Kubernetes clusters. Currently only workload ingress policy is supported.
+    
 
-    **Limitations**
+
+    #### Limitations
 
     - **No upgrades**: Calico v3.0.0 ends support for etcd version 2. Existing customers must migrate their data to etcd version 3. The alpha release does not provide migration capabilities, nor does it support upgrades. We plan to add migration and upgrade support in the GA release.
 
-    - **Integrates only with Kubernetes and host endpoints**: the Docker, OpenStack, OpenShift, Mesos, DC/OS, and rkt orchestrators have not been tested and are not supported. (Calico v3.0.0 still supports Docker and rkt containers.)
+    - **Integrates only with Kubernetes and host endpoints**: the Docker, OpenStack, OpenShift, Mesos, DC/OS, and rkt orchestrators have not been tested and are not supported. (Calico v3.0.0 still supports Docker and rkt containers.) We plan to resume support for the Docker, OpenStack, OpenShift, Mesos, and DC/OS orchestrators in a future release.
 
     - **Lack of `calicoctl` data validation**: `calicoctl` does not perform as much validation on data, increasing the potential for bad data. Use caution when entering data via `calicoctl`.
 
-    - **No support for BGP route reflection**: large deployments that require [BGP route reflection](https://docs.projectcalico.org/v3.0/usage/routereflector/bird-rr-config) are not supported.
+    - **BGP route reflection not supported**: large deployments that require [BGP route reflection](https://docs.projectcalico.org/v3.0/usage/routereflector/bird-rr-config) are not supported.
```

Comment:
sorry didn't see this earlier, but we also plan to resume support for the route reflector after this release

Reply:
OK good point.